snooty.toml

```
@@ -20,3 +20,15 @@ stable-api = "Stable API"
 node-driver-long = "MongoDB Node.js Driver"
 node-driver-short = "Node.js Driver"
 atlas = "MongoDB Atlas"
+
+[[banners]]
+targets = ["motor.txt"]
+variant = "warning"
+value = """
+    Motor has been sunset in favor of the PyMongo Async API in the PyMongo driver. \
+    Motor will be deprecated one year after the production release of the PyMongo Async API. \
+    No further bug fixes will be released after May 14th, 2026, with the exception of critical bug fixes, which will be released until May 14th, 2027. \
```

I think this can be more susinct:
```suggestion
    Motor has been replaced by the PyMongo Async API, and will be deprecated on May 14th, 2026.\
    No further bug fixes will be released after May 14th, 2026, with the exception of critical bug fixes, which will be released until May 14th, 2027. \
```
I'm not sure how much control you have over formatting, but if possible, there should be a line break at the end of this section.

---

I agree, but I can't get the banner to add any new lines